charts/README.md

```diff
@@ -0,0 +1,19 @@
+# Quantum serverless deployment
+Infrastructure part of quantum serverless project. Here you will find all the resources required to set up the correct environment to run the project.
+
+### Requirements
+- Helm >= 3.10
+
+## Technologies
+
+With these tools you will set up the infrastructure. Below the infrastructure the project makes use of [Ray](https://www.ray.io/) as the main framework to scale
+the different Python executions that you can send to your [k8s](https://kubernetes.io/).
+
+## Helm
+In this folder you will find the main charts to set up your k8s cluster and the services that this project uses. There are 6 main services:
+- **Jupyter**: this configuration deploys in your cluster the service that provides you with a notebook to work easily with the project.
```

Comment:
maybe note this (and the granfana / prometheus ones) as optional?

Reply:
Done in [2d80909](https://github.com/Qiskit-Extensions/quantum-serverless/pull/868/commits/2d80909758a1ce80c98820cae07dc2c46f37e2f5)